frontend/lib/core/utils/size_utils.dart

```diff
@@ -0,0 +1,122 @@
+// This code provides responsiveness – it is generated by DhiWise
+
+import 'package:flutter/material.dart';
+
+// This is where the magic happens.
+// This functions are responsible to make UI responsive across all the mobile devices.
+
+Size size = WidgetsBinding.instance.window.physicalSize /
+    WidgetsBinding.instance.window.devicePixelRatio;
+
+// Caution! If you think these are static values and are used to build a static UI,  you mustn’t.
```

Comment:
Love this comments, if theyre autogenerated, its hell of a feature from this dhiWise